src/aznfswatchdog

```diff
@@ -122,22 +149,57 @@ reconcile_conntrack()
             if [[ "$line" =~ $matchstr ]]; then
                 l_sport=${BASH_REMATCH[1]}
                 l_dport=${BASH_REMATCH[2]}
-                reconcile_conntrack_one $l_ip $l_sport $l_dport $l_nfsip
+                reconcile_conntrack_one $l_ip $l_sport $l_dport $l_nfsip "SYN_SENT"
+            fi
+        done
+    fi
+
+    IFS=$'\n' output111=$(conntrack -L -p tcp -d $l_ip --dport 111 --state ESTABLISHED 2>/dev/null)
+    IFS=$'\n' output2048=$(conntrack -L -p tcp -d $l_ip --dport 2048 --state ESTABLISHED 2>/dev/null)
+    output="$output111"$'\n'"$output2048"
+    if [ -n "$output" ]; then
+        for line in $output; do
+            # XXX Remove this log after running for few days.
+            vecho "$line"
+
+            matchstr=".* ESTABLSIHED src=([0-9]+\.[0-9]+\.[0-9]+\.[0-9]+) dst=$l_ip sport=([0-9]+) dport=([0-9]+) [UNREPLIED].*"
+            if [[ "$line" =~ $matchstr ]]; then
+                l_sysip=${BASH_REMATCH[1]}
+                l_sport=${BASH_REMATCH[2]}
+                l_dport=${BASH_REMATCH[3]}
+                reconcile_conntrack_two $l_ip $l_sport $l_dport $l_sysip "ESTABLISHED_UNREPLIED"
             fi
         done
     fi
 
     #
     # Any port that is not stuck now, means it's recovered since the last time and can be removed
-    # from the global cthash_synsent map.
+    # from the global cthash_stuck map.
     #
-    for key in "${!cthash_synsent[@]}"; do
-        if [ ! -v cthash_synsent_now[$key] ]; then
-            unset cthash_synsent[$key]
+    for key in "${!cthash_stuck[@]}"; do
+        if [ ! -v cthash_stuck_now[$key] ]; then
+            unset cthash_stuck[$key]
         fi
     done
 }
 
+#
+# Function for running stat for mountpoint so that everytime DNAT rule is updated, it is 
+# used to make sure to send atleast one packet that matches the DNAT rule.
+# This will eventually make sure the conntrack entry created always for communication.
+#
```

Comment:
`#`
`#` Function for running stat for mountpoint so that everytime DNAT rule is updated, it is 
`#` used to make sure to send atleast one packet that matches the DNAT rule.
`#` This will make sure that the connection gets a TCP reset and outstanding NFS RPC requests
`#` are retransmitted right away w/o waiting for the 1 min timeout.
`#`